deps/jerry/jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-uint8array-prototype.c

```diff
@@ -40,4 +54,131 @@
  * @}
  */
 
+/**
+ * The Uint8Array.prototype object's 'slice' routine
+ *
+ * See also:
+ *          ECMA-262 v5, 15.4.4.10
+ *
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value.
+ */
+static ecma_value_t
+ecma_builtin_uint8array_prototype_object_slice (ecma_value_t this_arg, /**< 'this' argument */
+                                           ecma_value_t arg1, /**< start */
```

Comment:
Use `begin`, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/slice

Reply:
Never mind, the standard use the word "start", let's follow that :)